libsolidity/interface/FileReader.cpp

```diff
@@ -35,6 +36,58 @@ using std::string;
 namespace solidity::frontend
 {
 
+FileReader::FileReader(
+	boost::filesystem::path _basePath,
+	FileSystemPathSet _allowedDirectories
+):
+	m_basePath(move(_basePath)),
+	m_allowedDirectories(move(_allowedDirectories)),
+	m_sourceCodes()
+{
+	solAssert(m_basePath.empty() || boost::filesystem::is_directory(m_basePath), "");
+
+	// TMP: Make sure this does not add "" to allowed directories with default base path
+	if (!contains(m_allowedDirectories, m_basePath))
+		// TMP: This might change the case with trailing /, see comment in allowDirectory()
+		allowDirectory(m_basePath);
+
+	for (boost::filesystem::path directory: _allowedDirectories)
+		allowDirectory(move(directory));
```

Comment:
Hmm, `_allowedDirectories` is moved to `m_alloweDirectories` above so this shouldn't even work, but even if it does it just duplicates the entires?

Reply:
Good catch. Should have been `_allowedDirectories`.